.gitignore

```diff
@@ -10,7 +11,4 @@ system-test/*key.json
 *.lock
 .DS_Store
 package-lock.json
-.vscode
-yarn.lock
 __pycache__
-.idea
```

Comment:
can we add one line here? :)

Reply:
I added it :) The "no new line" thingy is red, not green! :)

Reply:
Thank you!